packages/shopping/src/__tests__/acceptance/authentication.acceptance.ts

```diff
@@ -12,9 +12,9 @@ import _ from 'lodash';
 import {Suite} from 'mocha';
 import {ShoppingApplication} from '../..';
 import {PasswordHasherBindings, UserServiceBindings} from '../../keys';
-import {User} from '../../models';
+import {NewUser, User} from '../../models';
```

Comment:
`NewUser` -> `User`, `User` -> `BaseUser`?

Doing something like `new NewUser()` looks and feels odd.

Reply:
This is just a quick suggestion, feel free to share you thoughts on it.

Reply:
So NewUser extends User to add password property thus we can call it UserPassword since UserCredentials is already taken

As for MyUserService not sure what we can change it to but I do agree the property name could be changed to userService instead of myUserService. Similarly, the aNewUser properties can be changed to user

Reply:
So here is my suggestion:

1. MyUserService could be renamed to UserManagementService
2. NewUser to User
3. User to BaseUser

side note: not sure whether we will need to update controllers and repositories to use user-management... too?

I will then have to update the README to reflect the same if you are happy with this

Reply:
Why not just `UserService`? Let's update whatever is applicable.

Reply:
There is already user service from @loopback/authentication hence linting errors if the name is repeated

Reply:
I see, let's go with `UserManagementService` then.

Reply:
Done the changes accordingly. I wasnt sure what to call NewUser since changing it to User would require lots of changes particularly on the repositories so I changed it to UserPassword. Your thoughts?